core/src/autogluon/core/models/ensemble/fold_fitting_strategy.py

```diff
@@ -760,4 +777,4 @@ def __init__(self, **kwargs):
 
     def _sync_model_artifact(self, local_path, model_sync_path):
         bucket, path = s3_path_to_bucket_prefix(model_sync_path)
-        download_s3_folder(bucket, path, local_path)
+        download_s3_folder(bucket, path, local_path, keep_root_dir=True, error_if_exists=False)
```

Comment:
1. `keep_root_dir` doesn't exist.
2. Use keyword args

Reply:
Something probably went wrong when I was building the wheel and testing this functionality on ray cluster. This should be caught. I'll fix it and re-test it.